apps/web/src/jobs/index.ts

```diff
@@ -1,11 +1,24 @@
 import { setupJobs } from '@latitude-data/jobs'
+import env from '$/env'
 
-import env from '../env'
+/**
+ * In theory Nextjs should not pre-render pages that export
+ *   export const dynamic = 'force-dynamic'
```

Comment:
This is kind of a lie. I'm avoiding calling Redis while building. But not sure this is working tbh. Again only an issue for production. Let's keep this way for now